moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -25,29 +25,27 @@ target_link_libraries(${MOVEIT_LIB_NAME}
   ${BULLET_LIBRARIES}
```

Comment:
Can you add `Bullet` to the `ament_target_dependencies` block above? The variables seem to match the right pattern, so this should work.

Reply:
Isn't that only necessary for ros2 dependencies?

Reply:
> Isn't that only necessary for ros2 dependencies?

no, it works for all kinds of packages as long as they provide the right variables (_LIBRARIES, _INCLUDE_DIRS, ...).